src/client/app/components/DateRangeComponent.tsx

```diff
@@ -35,6 +37,14 @@ export default function DateRangeComponent() {
 		dispatch(changeSliderRange(dateRangeToTimeInterval(value)));
 	};
 
+	// Provide a default date range if the interval is unbounded to prevent error display
+	// Default to last 30 days when unbounded
+	const displayInterval = queryTimeInterval.getIsBounded()
+		? queryTimeInterval
+		: new TimeInterval(
+			moment().subtract(30, 'days').startOf('day'),
```

Comment:
In src/client/app/components/ThreeDIntervalControlsComponent.tsx there is DEFAULT_3D_DAYS. How does this relate to the 30 days here?